dev/django-admin.ts

```diff
@@ -0,0 +1,1293 @@
+      },
+      {
+        name: "--plan",
+        description:
+          "Shows a list of the migration actions that will be performed.",
+      },
+      {
+        name: "--run-syncdb",
+        description: "Creates tables for apps without migrations.",
+      },
+      {
+        name: "--check",
+        description:
+          "Exits with a non-zero status if unapplied migrations exist.",
+      },
+      {
+        name: "--skip-checks",
+        description: "Skip system checks.",
+      },
+    ],
+    description:
+      "Updates database schema. Manages both apps with migrations and those without.",
+    args: [
+      {
+        name: "app_label",
+        description: "App label of an application to synchronize the state.",
+        isOptional: true,
+      },
+      {
+        name: "migration_name",
+        description:
+          'Database state will be brought to the state after that migration. Use the name "zero" to unapply all migrations.',
+        isOptional: true,
+      },
+    ],
+  },
+  {
+    name: "sendtestemail",
+    icon: DJANGO_ICON_URL,
+    options: [
+      {
+        name: "--managers",
+        description:
+          "Send a test email to the addresses specified in settings.MANAGERS.",
+      },
+      {
+        name: "--admins",
+        description:
+          "Send a test email to the addresses specified in settings.ADMINS.",
+      },
+      {
+        name: "--skip-checks",
+        description: "Skip system checks.",
+      },
+    ],
+    description:
+      "Sends a test email to the email addresses specified as arguments.",
+    args: [
+      {
+        name: "email",
+        description: "One or more email addresses to send a test email to.",
+        // This argument is not optional when no other arguments are supplied.
+        // But it is optional when either `--managers` or `--admins` are supplied as arguments,
+        // so keeping it as `isOptional` for now to keep things simple.
+        isOptional: true,
+        variadic: true,
+      },
+    ],
+  },
+  {
+    name: "shell",
+    icon: DJANGO_ICON_URL,
+    options: [
+      {
+        name: "--no-startup",
+        description:
+          "When using plain Python, ignore the PYTHONSTARTUP environment variable and ~/.pythonrc.py script.",
+      },
+      {
+        name: ["-i", "--interface"],
+        description:
+          'Specify an interactive interpreter interface. Available options: "ipython", "bpython", and "python"',
+        args: {
+          name: "{ipython,bpython,python}",
+          suggestions: ["ipython", "bpython", "python"],
+        },
+      },
+      {
+        name: ["-c", "--command"],
+        description:
+          "Instead of opening an interactive shell, run a command as Django and exit.",
+        args: {
+          name: "COMMAND",
+        },
```

Comment:
Is this a shell command? If so, add `isCommand: true` and then Fig will continue to provide autocompletion on this argument.
```suggestion
        args: {
          name: "COMMAND",
          isCommand: true,
        },
```

Reply:
Just tested this out and it looks like "command" is a bit of a misleading name for this (though it is the one Django uses).

It looks like this takes a python statement and evaluates it, so you can do something like `python manage.py shell -c 'print(2 + 2)'` and it will return `4`

but if you try and do something like `python manage.py shell -c 'echo "hello world"'` it throws a SyntaxError

So that means it should probably stay as it is?

Reply:
In this case I think it would be better to not use `isCommand` since it's not a shell command but a python command.